### PR TITLE
Rephrase protocols intro and add links

### PR DESCRIPTION
**Pull request by bpires**
This PR rephrases protocols general intro and includes references and links to articles and videos.

Related to issue #24

### CODE REVIEW

File: content/topics/protocols/index.md [hunk 1]
```diff
@@ -8,11 +8,32 @@ menu:
     name: Protocols
 ---
 
-## Protocols are what exactly?
+## What are protocols?
```
Comment (Rico-Smit) [inline content/topics/protocols/index.md:11]:
```suggestion
## What are protocols exactly?
```

Comment (Rico-Smit) [reply content/topics/protocols/index.md:11]: I like the "exactly" in the title as we will be diving in to specifics. Thoughts?

**Review by Rico-Smit — CHANGES_REQUESTED** [review]
See comments in title section, The rest looks good and I like how you phrased it.

File: content/topics/protocols/index.md (hunk 1 above)
Comment (bpires) [reply content/topics/protocols/index.md:11]: LGTM!

**Review by Rico-Smit — APPROVED** [review]

**Comment by Rico-Smit** [comment]
Sounds good, all honesty i didn't see the email and approved the merge
already. But if their are any required changes i won't mind fix it.

File: content/topics/protocols/index.md [hunk 2]
```diff
@@ -8,11 +8,32 @@ menu:
     name: Protocols
 ---
 
-## Protocols are what exactly?
+## What are protocols exactly?
 
-A protocol is a set of rules and guidelines for communicating data. Rules are defined for each step and process during communication between two or more computers. Here we will cover many of these protocols from Internet Protocol(IP) to Post Office Protocol 3(POP3).
+Internet Protocols, or Network Protocols, are a set of rules and guidelines for communicating data between computers connected to the Internet.
```
Comment (schalkneethling) [inline content/topics/protocols/index.md:13]: Hey @Rico-Smit and @bpires,

I would rephrase this a little. Either:

> Internet Protocols, or Network Protocols, are rules and guidelines for communicating data between computers connected to the Internet.

or

> Internet Protocols, or Network Protocols, are rules and guidelines for sending data between computers connected to the Internet.

**Comment by schalkneethling** [comment]
Great work on this! 🎉
